test/jasmine/tests/sankey_test.js

```diff
@@ -482,6 +482,28 @@ describe('sankey tests', function() {
                 .catch(failTest)
                 .then(done);
         });
+
+        it('should not show labels if hovermode is false', function(done) {
+            var gd = createGraphDiv();
+            var mockCopy = Lib.extendDeep({}, mock);
+
+            function _hover(px, py) {
+                mouseEvent('mousemove', px, py);
+                mouseEvent('mouseover', px, py);
+                Lib.clearThrottle();
+            }
```

Comment:
Can we push this to an outer scope so we're not just repeating the function from above?